File: kaesar.js
const alphabet = "ێیەھۆونمڵلگکقڤفغعشسژزڕردخحچجتپبائ";
let output ="";

function encrypt(text, key) {
  for (let char of text) {
    if (alphabet.includes(char)) {
      let position = alphabet.indexOf(char);
      let newPosition = (position + key) % 33;
      let newChar = alphabet[newPosition];
      output += newChar;
    } else {
      output += char;
    }
  }
  return output;
}
function decrypt(text, key) {
  for (let char of text) {
    if (alphabet.includes(char)) {
      let position = alphabet.indexOf(char);
      let newPosition = (position - key) % 33;
      let originalChar = alphabet[newPosition];
      output += originalChar;
    } else {
      output += char;
    }
  }
  return output;
} 
//شتێک بنووسە
const text = "بردنەوە";
const key = 3; //کلیل
console.log(encrypt(text, key)); // بۆ ئینکریپت
//console.log(decrypt(text, key)); //بۆ دیکریپت

File: kaesar.py
alphabet = "ئابپتجچحخدرڕزژسشعغفڤقکگلڵمنوۆھەیێ"

def encrypt(text, key):
  output = ""
  for char in text:
    if char in alphabet:
      position = alphabet.find(char)
      newposition = (position + key) % 33
      newChar = alphabet[newposition]
      output += newChar
    else:
      output += char
  return output

def decrypt(text, key):
  output = ""
  for char in text:
    if char in alphabet:
      position = alphabet.find(char)
      newposition = (position - key) % 33
      originalChar = alphabet[newposition]
      output += originalChar
    else:
      output += char
  return output

if __name__ == "__main__":
  k = int(input("٠ بنووسە بۆ ئینکریپت یاخود ١ بۆ دیکریپت کردن: "))
  text = input("شتێک بنووسە: ")
  key = int(input("کلیل: "))

  if k == 0: print(encrypt(text, key))
  elif k == 1: print(decrypt(text, key))
  else: print("دووبارە هەوڵ بدەرەوە")